src/main/webapp/view/discussion/discussion-manager.js

```diff
@@ -88,22 +89,24 @@ export default class DiscussionManager {
 
   /**
    * Posts `content` as a new root comment at `timestampMs` with the specified
-   * `type`.
+   * `type` and `transcriptLineId`.
    */
-  async postRootComment(content, timestampMs, type) {
+  async postRootComment(content, timestampMs, type, transcriptLineId) {
     await this.postComment(content, {
       [DiscussionManager.#PARAM_TIMESTAMP]: timestampMs,
       [DiscussionManager.#PARAM_TYPE]: type,
+      [DiscussionManager.#PARAM_TRANSCRIPT_LINE]: transcriptLineId,
     });
   }
 
   /**
-   * Posts `content` as a reply to `parentId`.
+   * Posts `content` as a reply to `parentId` with the specified `transcriptId`.
```

Comment:
'transcriptId' is a type.the field is transcriptLineId.

Reply:
ah, whoops, fixed!